tests/tests_unit/test_api/test_functions.py

```diff
@@ -855,14 +855,12 @@ def test_retrieve_schedules(self, mock_function_schedules_retrieve_response, cog
         res = cognite_client.functions.schedules.retrieve(id=SCHEDULE_WITH_FUNCTION_EXTERNAL_ID["id"])
         assert isinstance(res, FunctionSchedule)
         expected = mock_function_schedules_retrieve_response.calls[0].response.json()["items"][0]
-        expected.pop("when")
         assert expected == res.dump(camel_case=True)
 
     def test_list_schedules(self, mock_filter_function_schedules_response, cognite_client):
         res = cognite_client.functions.schedules.list()
         assert isinstance(res, FunctionSchedulesList)
         expected = mock_filter_function_schedules_response.calls[0].response.json()["items"]
-        expected[0].pop("when")
```

Comment:
That is a very good question. Now I'm starting to wonder if there were actually good reasons for it, but which have since been forgotten... But let's try adding it - we can remove if we discover issues with it.